cypress/e2e/conversations.cy.js

```diff
@@ -1,267 +0,0 @@
-//  /// <reference types="cypress" />
-import scores from "../fixtures/postScores.json";
-
-var faker = require("faker");
-
-let session_Id;
-let set_one_quizId;
-let user;
-let accessToken;
-const invalidInvitedUsername = "Must provide a name that is up to 20 characters long.";
-const missingJSONBodyMessage = "Must provide a JSON body with the name of the invited user.";
-const SESSION_UUIDNotInDBMessage = "SESSION_UUID is not in the db.";
-const uuidFormatChecker_LowerCase = /[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/;
-const uuidFormatChecker_UpperCase = /[0-9A-F]{8}-[0-9A-F]{4}-[0-9A-F]{4}-[0-9A-F]{4}-[0-9A-F]{12}/;
-const createdSuccessfulMessage = "conversation created";
-let requestBody;
-let invitedUserName;
-
-describe("'/conversation' endpoint", () => {
-    beforeEach(() => {
-        cy.sessionEndpoint().should((response) => {
-            session_Id = response.body.sessionId
-        }).then(() => {
-            cy.scoresEndpoint(scores, session_Id).should((response) => {
-                set_one_quizId = response.body.quizId;
-            }).then(() => {
-                user = {
-                    "firstName": faker.name.firstName(),
-                    "lastName": faker.name.lastName(),
-                    "email": faker.internet.email(),
-                    "password": `@7${faker.internet.password()}`,
-                    "quizId": set_one_quizId
-                };
-                expect(user.firstName).to.be.a('string')
-                cy.registerEndpoint(user).should((response) => {
-                    expect(response.status).to.equal(201);
-                    accessToken = response.body.access_token;
-                    expect(response.body.access_token).to.be.a('string')
-                });
-            });
-        });
-    });
-
-    it("should create a conversation", () => {
-        //Generate a username to be invited
-        requestBody = {
-            "invitedUserName": faker.name.firstName()
-        };
-        expect(requestBody).to.be.an("object");
-        expect(requestBody.invitedUserName).to.be.a("string");
-
-        //Create a conversation and test the response body
-        cy.conversationEndpoint(requestBody, accessToken, session_Id).should((response) => {

```

Comment:
I'd suggest deleting `conversationEndpoint` in `cypress/support/commands.js`, since it's no longer used anywhere in the codebase.